.gitignore

```diff
@@ -1,3 +1,5 @@
 manifests/deis-router-rc.tmp.yaml
+rootfs/bin/router
 rootfs/opt/router/sbin/router
+rootfs/opt/router/sbin/router.*
```

Comment:
`upx` created some temp binaries apparently. I had `router.upx` and `router.000` binaries there somehow while testing this PR.